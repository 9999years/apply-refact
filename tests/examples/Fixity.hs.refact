[("tests/examples/Fixity.hs:1:10: Warning: Move brackets to avoid $\nFound:\n  h <$> (f $ g $ h t)\nWhy not:\n  h <$> f (g $ h t)\n",[Replace {rtype = Expr, pos = SrcSpan {startLine = 1, startCol = 16, endLine = 1, endCol = 29}, subts = [("a",SrcSpan {startLine = 1, startCol = 17, endLine = 1, endCol = 18}),("b",SrcSpan {startLine = 1, startCol = 21, endLine = 1, endCol = 28})], orig = "a (b)"}])]
