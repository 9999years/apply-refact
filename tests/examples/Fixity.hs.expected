simple = h <$> f (g $ h t)

simple = h <$> (f :+ g :+ h t)

simple = h <$> (f `elem` h `elem` h)
